extensions/wikia/Recirculation/js/views/mixedFooter.js

```diff
@@ -99,17 +100,28 @@ define('ext.wikia.recirculation.views.mixedFooter', [
 	}
 
 	function setupTracking() {
+		var shouldSendAdditionalData = window.Wikia.AbTest.inGroup('RECOMMENDATION_SERVICE', 'EXPERIMENTAL');
+
 		tracker.trackImpression('footer');
 
 		$mixedContentFooter.on('click', '[data-tracking]', function () {
 			var $this = $(this),
 				labels = $this.data('tracking').split(','),
-				href = $this.attr('href');
+				href = $this.attr('href'),
+				additionalParams;
+
+			if (shouldSendAdditionalData && labels.indexOf('wiki-article') !== -1) {
+				additionalParams = {
+					recommendation_request_id: recommendedContent.getRequestId(),
+					item_id: $this.attr('data-item-id'),
+					item_type: 'wiki_article'
```

Comment:
> item_type (possible values are: wiki_article, wiki, ns_article etc. for now only wiki_article is used).

I decided to hardcode it for now, since the approach may change later